code/modules/surgery/ear_surgery.dm

```diff
@@ -73,3 +73,15 @@
 		)
 		display_pain(target, "You feel a visceral stabbing pain right through your head!") // dunno who can feel pain w/o a brain but may as well be consistent.
 	return FALSE
+
+/datum/surgery/ear_surgery/mechanic
+	name = "Ear surgery"
+	requires_bodypart_type = BODYTYPE_ROBOTIC
+	target_mobtypes = list(/mob/living/carbon/human) 
+	steps = list(
+		/datum/surgery_step/mechanic_open,
+		/datum/surgery_step/open_hatch,
+		/datum/surgery_step/prepare_electronics,
+		/datum/surgery_step/fix_ears,
+		/datum/surgery_step/mechanic_close,
+	)
```

Comment:
modularize (just move to like `monkestation/code/modules/surgery/ear_surgery.dm`)